conversation_classification/kaggle/trainer/model.py

```diff
@@ -30,15 +43,25 @@
 MAX_LABEL = 2
 MAX_DOCUMENT_LENGTH = 500 # Max length of each comment in words
 
-# Model Params
-EMBEDDING_SIZE = 50 # Size of learned  word embedding
+CNNParams = namedtuple(
+  'CNNParams', ['WINDOW_SIZE', 'EMBEDDING_SIZE','POOLING_WINDOW', 'POOLING_STRIDE',
+                'N_FILTERS', 'FILTER_SHAPE1', 'FILTER_SHAPE2'])
+cnn_values = {'WINDOW_SIZE':20, 'EMBEDDING_SIZE':20, 'POOLING_WINDOW':4,
+              'POOLING_STRIDE':2, 'N_FILTERS':10}
+cnn_values['FILTER_SHAPE1'] = [cnn_values['WINDOW_SIZE'], cnn_values['EMBEDDING_SIZE']]
+cnn_values['FILTER_SHAPE2'] = [cnn_values['WINDOW_SIZE'], cnn_values['N_FILTERS']]
+CNN_PARAMS = CNNParams(**cnn_values)
+
+BOWParams = namedtuple('BOWParams', ['EMBEDDING_SIZE'])
+BOW_PARAMS = BOWParams(EMBEDDING_SIZE = 20)
```

Comment:
I'm not sure what the purpose is of using namedtuples instead of just the underlying dictionary?